tests/routes/case.test.js

```diff
@@ -137,6 +168,122 @@ describe('/cases route', () => {
+          createdCasesAndContacts.push({
+            contact: createdContact,
+            case: createdCase,
+          });
+        }
+      });
+
+      afterAll(async () => {
+        return Promise.all([
+          Case.destroy({ where: { id: createdCasesAndContacts.map(ccc => ccc.case.id) } }),
+          Contact.destroy({ where: { id: createdCasesAndContacts.map(ccc => ccc.contact.id) } }),
+        ]);
+      });
+
+      // eslint-disable-next-line jest/expect-expect
+      each([
+        {
+          description:
+            'should return all cases for account when no helpline, limit or offset is specified',
+          listRoute: `/v0/accounts/${accounts[0]}/cases`,
+          expectedCasesAndContacts: () =>
+            createdCasesAndContacts
+              .filter(ccc => ccc.case.dataValues.accountSid === accounts[0])
+              .sort((ccc1, ccc2) => ccc2.case.id - ccc1.case.id),
+          expectedCount: 5,
+        },
+        {
+          description: 'should return all cases for account & helpline when helpline is specified',
+          listRoute: `/v0/accounts/${accounts[0]}/cases?helpline=${helplines[1]}`,
+          expectedCasesAndContacts: () =>
+            createdCasesAndContacts
+              .filter(
+                ccc =>
+                  ccc.case.dataValues.accountSid === accounts[0] &&
+                  ccc.case.helpline === helplines[1],
+              )
+              .sort((ccc1, ccc2) => ccc2.case.id - ccc1.case.id),
+          expectedCount: 1,
+        },
+        {
+          description: 'should return first X cases when limit X is specified',
+          listRoute: `/v0/accounts/${accounts[0]}/cases?limit=3`,
+          expectedCasesAndContacts: () =>
+            createdCasesAndContacts
+              .filter(ccc => ccc.case.dataValues.accountSid === accounts[0])
+              .sort((ccc1, ccc2) => ccc2.case.id - ccc1.case.id)
+              .slice(0, 3),
+          expectedCount: 5,
+        },
+        {
+          description:
+            'should return X cases, starting at Y when limit X and offset Y are specified',
+          listRoute: `/v0/accounts/${accounts[0]}/cases?limit=2&offset=1`,
+          expectedCasesAndContacts: () =>
+            createdCasesAndContacts
+              .filter(ccc => ccc.case.dataValues.accountSid === accounts[0])
+              .sort((ccc1, ccc2) => ccc2.case.id - ccc1.case.id)
+              .slice(1, 3),
+          expectedCount: 5,
+        },
+        {
+          description:
+            'should return remaining cases, starting at Y when limit X and offset Y are specified',
+          listRoute: `/v0/accounts/${accounts[0]}/cases?offset=2`,
+          expectedCasesAndContacts: () =>
+            createdCasesAndContacts
+              .filter(ccc => ccc.case.dataValues.accountSid === accounts[0])
+              .sort((ccc1, ccc2) => ccc2.case.id - ccc1.case.id)
+              .slice(2),
+          expectedCount: 5,
```

Comment:
Here de description says `limit` and `offset` are specified, but looking at the URL we're specifying only the `offset`